param/motion_planner.yaml

```diff
@@ -41,7 +41,15 @@ roomba_max_end_task_dist: .1
 k_term_tracking_x: .35
 # p-term for Y direction when tracking roombas 
 k_term_tracking_y: .35
-# p-term for holding Z height
-k_term_hold_z: 1.25
-#maximum z error for heigth holder
+# p-term for height holder helper for Z height
+k_term_hold_z: .90
```

Comment:
Ok fixed, but this is the p-term for the z height in the height holder helper. Position holder task and height holder helper use different p-terms for z-height